docs/templates/eia861_child.rst.jinja

```diff
@@ -18,22 +28,81 @@ details, see the `official EIA-861 page <{{ source.path }}>`__
 {% endblock %}
 
 {% block availability %}
+PUDL incorporates all annual EIA-861 data.
+Earlier data (1990-2000) exists but has not been incorporated into PUDL yet.
```

Comment:
It's a little strange to say that we incorporate all annual data except data from 1990-2000. We should reword this. @zaneselvans is there a reason we don't incorporate the earlier data? Is it a different format or something?

Reply:
There's much less information for each year, the structure is quite different, and in general there seems to be less interest in the data the further back in time we go.